examples/navigation/epics/adminAccess.js

```diff
@@ -8,9 +8,9 @@ export default function adminAccess(action$) {
     // If you wanted to do an actual access check you
     // could do so here then filter by failed checks.
     .delay(2000)
-    .mergeMap(() => Observable.merge(
-      Observable.of(accessDenied()),
-      Observable.timer(2000)
-        .map(() => push('/'))
-    ));
+    .mergeMap(() =>
+      Observable.of(push('/'))
+       .delay(2000)
+       .startWith(accessDenied())
```

Comment:
The reason I was partial (when originally writing up the example PR) to the `merge` option is because it reads well top to bottom. I'm no Rx ninja so for all I know using the `startWith` is more idiomatic. If that's the case I wonder what's more clear to beginners (considering that many devs will be introduced to Rx via `redux-observable`).

I don't really have a lot of experience teaching devs rx but maybe @jayphelps or @blesh can shed some light here.